kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java

```diff
@@ -194,13 +208,44 @@ public KModuleDeploymentUnit setup(DeploymentService ds, String artifactId) {
         return unit;
     }
 
-    protected void cleanup(DeploymentService ds, KModuleDeploymentUnit unit) {
-        if (ds!=null) {
+    protected void cleanup(DeploymentService ds, KModuleDeploymentUnit unit, String artifactId) {
+        abortAllProcesses();
+        if (ds!=null && unit!=null) {
             ds.undeploy(unit);
         }
+        if (kieServicesClient != null) {
+            kieServicesClient.disposeContainer(artifactId);
+        }
         System.clearProperty(MESSAGE_MAPPING_PROPERTY);
         System.clearProperty(SIGNAL_MAPPING_PROPERTY);
     }
+    
```

Comment:
I think you can make use of RuntimeDataService Collection<ProcessInstanceDesc> getProcessInstances(List<Integer> states, String initiator, QueryContext queryContext); method to retrieve all active process instance, so there is not need to use rest api

Reply:
Thanks @fjtirado! I haven't seen it, changed